client/onboarding-prototype/steps/loading.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { addQueryArgs } from '@wordpress/url';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { useOnboardingContext } from '../context';
+import { EligibleData, EligibleResult } from '../types';
+import { fromDotNotation } from '../utils';
+
+const Loading: React.FC = () => {
+	const { data } = useOnboardingContext();
+
+	const isEligibleForPo = async () => {
+		if (
+			! data.country ||
+			! data.business_type ||
+			! data.annual_revenue ||
+			! data.go_live_timeframe
+		) {
+			return false;
+		}
+		const businessDetails: EligibleData = {
+			business: {
+				country: data.country,
+				type: data.business_type,
+				mcc: 'computers_peripherals_and_software', // TODO GH-4853 add MCC from onboarding form
+				annual_revenue: data.annual_revenue,
+				go_live_timeframe: data.go_live_timeframe,
+			},
+		};
+		const eligibleResult = await apiFetch< EligibleResult >( {
+			path: '/wc/v3/payments/onboarding/router/po_eligible',
+			method: 'POST',
+			data: businessDetails,
+		} );
+
+		return 'eligible' === eligibleResult.result;
+	};
+
+	const handleComplete = async () => {
+		const { connectUrl } = wcpaySettings;
+		// TODO GH-5476 prefill the data for full KYC with addQueryArgs( connectUrl, {prefill: fromDotNotation( data ),} )
+		// that needs server tweaks first.
+		const resultUrl = ( await isEligibleForPo() )
+			? addQueryArgs( connectUrl, {
+					progressive: fromDotNotation( data ),
+			  } )
+			: connectUrl;
+		window.location.href = resultUrl;
+	};
+
+	useEffect( () => {
+		handleComplete();
+		// We only want to run this once, so we disable the exhaustive deps rule.
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [] );
+
+	// TODO implement in GH-4746 (Full page design for PO)
```

Comment:
I wasn't able to find a loading bar component, so I added it to #4744.

Reply:
Thanks! Fixed comment in https://github.com/Automattic/woocommerce-payments/pull/5779/commits/20e1c1699803c51e24eafef8001a15141e6feba2.